src/main/java/mate/academy/hibernate/relations/service/impl/CountryServiceImpl.java

```diff
@@ -1,16 +1,24 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import mate.academy.hibernate.relations.dao.CountryDao;
 import mate.academy.hibernate.relations.model.Country;
 import mate.academy.hibernate.relations.service.CountryService;
 
 public class CountryServiceImpl implements CountryService {
+    private final CountryDao countryDao;
+
+    public CountryServiceImpl(CountryDao countryDao) {
+        this.countryDao = countryDao;
+    }
+
     @Override
     public Country add(Country country) {
-        return null;
+        return countryDao.add(country);
     }
 
     @Override
     public Country get(Long id) {
-        return null;
+        return countryDao.get(id).orElseThrow(() ->
+                new RuntimeException("Actor not found"));
```

Comment:
Check when you copy-paste code. 
```suggestion
                new RuntimeException("Country not found"));
```
The same as in actor class